TickSpec.Tests/FeatureParserTest.fs

```diff
@@ -16,20 +16,30 @@ let private verifyParsing (lines: string[]) (expected: FeatureSource) =
 let private verifyLineParsing (lines: string[]) (expected: LineType list) =
     let lineParsed =
         lines
-        |> Seq.map (fun line ->
+        |> Seq.mapi (fun lineNumber line -> (lineNumber + 1, line))
```

Comment:
I dont think the `mapi >> map` makes things more legible - better to collapse to a single `mapi`

Reply:
Makes sense.